WRITING_DOCS.md

```diff
@@ -9,8 +9,6 @@
   - [Divider](#divider)
   - [Grid](#grid)
   - [Mini-Card](#mini-card)
```

Comment:
Also, when you shrink the "Getting Started" page to the point where the side-nav doesn't fit, they exhibit different behaviors.
The original site adds a hamburger menu at the top (good), the new site doesn't add this, making navigation difficult.
![image](https://github.com/icerpc/icerpc-docs/assets/25963603/f220d93d-7868-49ee-93f4-b5335afccb7c)
It looks like the new site still reserves the space for the menu, even if it doesn't actually show it.

Reply:
This seems to happen on every page, not just getting started,